lib/ethereumex/config.ex

```diff
@@ -2,39 +2,26 @@ defmodule Ethereumex.Config do
   @moduledoc false
   alias Ethereumex.IpcServer
 
-  def setup_children() do
-    setup_children(client_type())
-  end
-
-  defp setup_children(:ipc) do
-    path = Enum.join([System.user_home!(), ipc_path()])
+  def setup_children(type \\ client_type()) do
+    case type do
```

Comment:
I think it looked better with pattern matching on function signatures. please return it